docker-compose.yml

```diff
@@ -1,5 +1,6 @@
 services:
   pg:
+    # Todo: Do we care which version?
```

Comment:
pinned to 16.10 in the (very recently merged) PR https://github.com/communitiesuk/ukraine-sponsor-resettlement/pull/612

Reply:
```suggestion
```